src/navigation/Stepper/index.tsx

```diff
@@ -31,9 +31,10 @@ type Props = {
   activeStepIndex: number;
   error?: boolean;
   orientation: "vertical" | "horizontal";
+  startStepperAtZeroIndex?: boolean;
 };
 
-const Stepper = ({ steps, error, activeStepIndex, orientation }: Props) => {
+const Stepper = ({ steps, startStepperAtZeroIndex, error, activeStepIndex, orientation }: Props) => {
```

Comment:
this is too specific, can't we just make a prop `startIndex`?

Reply:
I would prefer to add a new attr here: steps: Array<{ id: string; dotLabel: string,  label: string }>. Perhaps with a better name.

Reply:
@alongoni wdyt?

Reply:
why? do you have a use case in mind where you want to pass custom label for a dot?

Reply:
not now, but it could be useful for others, think that we are not the only one using this lib now.

Reply:
I wouldn't do this until we explicitly are asked for it. It's possible to imagine a lot of possible use cases that might be handy for someone someday.

Also if you google `stepper custom dot label` there's only only question on stack overflow, so it's not the most popular use case and I wouldn't spend time on it